hooks/pre-push

```diff
@@ -125,7 +125,7 @@ else
               printf "${SUCCESS}\n"
         fi
 
-    run_cargo build
+        run_cargo build
 	run_cargo test
```

Comment:
Nit: Can you fix this to make the indentation consistent? Is the problem that some indents are tabs and some are spaces?